iina/PlayerWindowController.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  PlayerWindowController.swift
+//  iina
+//
+//  Created by Yuze Jiang on 2/15/20.
+//  Copyright © 2020 lhc. All rights reserved.
+//
+
+import Foundation
+
+class PlayerWindowController: NSWindowController {
+  unowned var player: PlayerCore
+  
+  var menuActionHandler: MainMenuActionHandler!
+  
+  var loaded = false
+  
+  init(playerCore: PlayerCore) {
+    self.player = playerCore
+    super.init(window: nil)
+  }
+
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+  
+  internal lazy var followGlobalSeekTypeWhenAdjustSlider: Bool = Preference.bool(for: .followGlobalSeekTypeWhenAdjustSlider)
```

Comment:
Does this need an annotation? Also, I'm not sure if we can change the name now but if we can I would consider making it shorter and more grammatical.

Reply:
Feel free to suggest a new name or something. What I did was just copy-paste from the two classes.

Reply:
Since we already have a preference key with this string, I think we should leave it for now :(